lifecycle-operator/apis/lifecycle/v1beta1/keptnworkloadversion_types_test.go

```diff
@@ -445,3 +445,17 @@ func TestKeptnWorkloadVersionList(t *testing.T) {
 	require.Equal(t, "obj1", got[0].GetName())
 	require.Equal(t, "obj2", got[1].GetName())
 }
+
+func TestKeptnWorkloadVersion_GetPromotionTasks(t *testing.T) {
+	w := KeptnWorkloadVersion{}
+
+	got := w.GetPromotionTasks()
+	require.Empty(t, got)
+}
+
+func TestKeptnWorkloadVersion_GetPromotionTaskStatus(t *testing.T) {
+	w := KeptnWorkloadVersion{}
+
+	got := w.GetPromotionTaskStatus()
+	require.Empty(t, got)
+}
```

Comment:
Let's make these tests part of this big test as all other functions are tested there https://github.com/keptn/lifecycle-toolkit/pull/3057/files#diff-c33da3306196915d27eb5f011eb4cd78b62f3f4b5ea7892b43933bf49590b573R15